src/ngraph/op/experimental/quantized_conv_bias.cpp

```diff
@@ -90,3 +90,146 @@ shared_ptr<Node> op::QuantizedConvolutionBias::copy_with_new_args(const NodeVect
                                                          new_args.at(3),
                                                          m_with_relu));
 }
+
+op::QuantizedConvolutionBiasAdd::QuantizedConvolutionBiasAdd(const shared_ptr<Node>& data_batch,
```

Comment:
It feels like overkill having a new op here.  @nbpatel tells me we support the f32 version of this op in the CPU backend via pattern matching.  Ideally, the same would be true for the int8 version.

Assume you get to keep the builder, but not the op.  What would the builder do to enable pattern matching in the CPU backend?  Would we need new (more elemental) quantized ops to make this work?

I'm concerned about feature creep.  So far, we are working just on TensorFlow, just on the CPU backend, and just for the few models we are targeting for our near-term quantization goals.  Worried that this sort or "complex" instruction set will break as we add more frameworks and backends later on.  Need to align as close as possible to nGraph "reduced" instruction set architecture.